deploy/infra/aws/terraform/cloud-native/cluster-infra/input.tf

```diff
@@ -233,3 +233,51 @@ variable "rds_db_port" {
   default     = 5432
   description = "The port on which the RDS instance accepts connections"
 }
+
+###################################################################
+#               Cloud-pipeline script deployment
+###################################################################
+
+variable "deployment_id" {
```

Comment:
for each parameter check and fix description of i, f.i. `deployment_id` has description as `Deployment Name` but what this is really means? how it will be used and where?